doc/whats_new.rst

```diff
@@ -196,6 +196,11 @@ Bug fixes
       :clas:`discriminant_analysis.LinearDiscriminantAnalysis` now returns
       correct results. By `JPFrancoia`_
 
+    - Fixed bug in :class:`linear_model.sgd_fast.SquaredEpsilonInsensitive`
+      where condition of if statement is incorrect when p - y > epsilon.
+      Affects :class:`linear_model.SGDClassifier` and :class:`linear_model.SGDRegressor`.
+      (`#6764 <https://github.com/scikit-learn/scikit-learn/pull/6764>`_). By `Wenhua Yang`_.
+
```

Comment:
Just a minor thing. Could you please add a link to your github profile at the bottom so that the name on this line generates a hyperlink to the profile page? Thanks.